busted/runner.lua

```diff
@@ -149,17 +149,41 @@ return function(options)
     })
     if not ok then
       io.stderr:write(appName .. ': failed running the specified helper (' ..
-                      cliArgs.helper .. '), error: ' .. err .. '\n')
+        cliArgs.helper .. '), error: ' .. err .. '\n')
```

Comment:
Style changes in lines not otherwise changing should be done in a separate commit. I'm happy to accept style changes but don't like them showing up in unrelated diffs largely because it makes code review hard, not just now in the PR but later when git bisecting or git blaming or anything like that.